draft-ietf-lake-edhoc.md

```diff
@@ -528,73 +625,14 @@ To provide forward secrecy in an even more efficient way than re-running EDHOC,
       PRK_4x3m = Extract( [ "TH_4", nonce ], PRK_4x3m )
 ~~~~~~~~~~~
 
```

Comment:
I think we can add some high level general sentence about how ephemeral keys used in "Ephemeral Public Keys" section.

The key derivation section also belongs better in the "Processing" part of "Message Formatting and Processing" in my opinion (as a subsection)